apiary.apib

```diff
@@ -652,11 +662,202 @@ A common question is whether or not to include multiple recipients on a single P
                }
             }
 
+## Shortlink [/shortlink]
+
+Shortlinks can be used when you want to deliver a unique link to a gift offer via your own distribution method.
+
+To generate a Shortlink call the `/campaign` endpoint with `delivery_type: "SHORTLINK"`.
+Many Shortlinks can be created in a single request and allow the recipient to choose from the offered gifts
+in the template or set in `brand_codes`.
+
+Shortlinks take a moment before they are fully created.
+In order to retrieve the link(s) you can call `/links/{id}` to check the status and once ready the API will provide the actual shortlink URL(s).
+
+After which you can distribute the links to your recipient via any method you choose. Then the recipient can open the link to claim their gift offer.
+
+### Deciding on Shortlinks vs In-App Delivery
+
+Since Shortlinks and In-App Delivery both serve very similar use cases, it often can be difficult to choose
+which delivery type to leverage. The following section outlines the differences between the two delivery types:
+
+#### Shortlinks:
+
+- The distribution to recipients is handled via link sharing by you.
+- Multiple gift options can be included to choose from instead of pre-selecting only one gift.
+- Gift offers may be canceled so long as the gift offer has not been redeemed or has not yet expired.
+- Multiple links may be generated in a single request.
+
+#### In-App Delivery:
+
+- Designed to be embedded into an existing app with minimal Giftbit branding and with no additional message as part of the gift.
+- The redemption experience is more streamlined with only a single gift offer that is pre-selected.
+- Users are automatically directed to the selected card without the need for additional steps.
+- All gift offers are marked redeemed. They cannot be canceled once created nor will they expire.
+- Each API request generates a single gift.
+
+### Obtaining the Shortlink
+
+Once a Shortlink request has been made, and if there are sufficient funds, it will take a short period of time before the links become available.
+We recommend polling the links endpoint every 5 minutes until it is ready.
```

Comment:
Do we recommend every 5 mins? That seems like a long time

Reply:
@tyrelh This was ported over from the older documentation so not sure if we need that long.   In testbed it is almost instant but that might not be representative.  

Going to test this in another environment and see if we could maybe change this to 1-2 minutes or as short as 30 seconds.

Reply:
@tyrelh Just tested this, generated a gift link and it took about 20-30 seconds.  We could probably change this to 2-3 minutes but 5 minutes is consistent with the API message:

 
![Screen Shot 2023-06-13 at 11 56 42 AM](https://github.com/Giftbit/Giftbit-API-Docs/assets/117672511/f0a06194-e450-440b-82f8-3511f197c325)

Reply:
Leaving it as is is fine if it's consistent with what were telling folks elsewhere.